docker-runner/docker-run.sh

```diff
@@ -7,4 +7,11 @@
 : "${CONTAINER_NAME:?}"
 
 set -e
-docker run --rm --link $CONTAINER_NAME:target -e ZGRAB_TARGET=target zgrab2_runner $@
+
+MOUNT=
```

Comment:
```suggestion
MOUNT=""
```
just so it's unambiguous.